MISC/SetGravityLevel.md

```diff
@@ -9,11 +9,15 @@ void SET_GRAVITY_LEVEL(int level);
 ```
 
 ```
-level can be from 0 to 3  
+This native adjusts the in-game gravity level by the given amount.
+```
+
+```
+Level can be set inbetween 0-3
```

Comment:
This is wrong. Gravity level goes from 0 to 3, by in-between you're stating 1, 2. Gravity is read from an array containing 4 IEEE754 values ranging from 0 to 9.8 when converted to float precision point.

```
void __fastcall setGravityLevelActual(unsigned int level)
{
  if ( level <= 3 )
  {
    gravityLevel = gravityArray[level];
    setGlobalVehicleGravityFactor(*&gravityLevel);
    updateDynamicsGravity();
    sub_7FF702D4E30C();
  }
}
```

The array:
```
int gravityArray [4] = { 0x411CCCCD, 0x4019999A, 0x3DCCCCCD, 0 };
```